src/ios/Utilities.m

```diff
@@ -12,10 +16,18 @@ + (NSString*)getApplicationTimestamp{
     return nil;
 }
 
-+ (NSDate*)getApplicationBuildTime{
-    NSString *appPath = [[NSBundle mainBundle] bundlePath];
-    NSDictionary *executableAttributes = [[NSFileManager defaultManager] attributesOfItemAtPath:appPath error:nil];
-    return [executableAttributes objectForKey:@"NSFileModificationDate"];
++ (NSDate*)getApplicationBuildTime{   
```

Comment:
Is it possible that this change fixed the problem, without the need for the appVersion check above?